src/SAML2/Provider/HostedIdentityProviderProcessor.php

```diff
@@ -544,16 +562,31 @@ protected function buildAuthnResponse(\SAML2_AuthnRequest $authnRequest)
         $authnResponseBuilder = new AuthnResponseBuilder();
 
         $state = $this->stateHandler->get();
+        $user = $this->stateHandler->getUser();
+        $nameIdValue =
```

Comment:
Good idea. I was expecting integrator using the event to configure the response. But this is more easier for them like this.